transport_nantes/mobilito/models.py

```diff
@@ -22,6 +35,37 @@ class MobilitoUser(models.Model):
     def __str__(self):
         return self.user.email
 
+    @property
+    def username(self):
+        return self._username or self.user.username
```

Comment:
There's really no way anyone coming at this code could possibly know why you are doing this.  Functions need documentation.

I don't think we wat to do this.  `User.username` is always set to a random string, but it would be quite counter-intuitive, if it's some day used for something, that it has a side-effect on publicly visible mobilito data.

Just initially generate a random string, as we do for user.  If the user wants to change it, they can.

Reply:
Another reason not to do this is that your integrity check is against `self._username` but you are displaying `self._username or self.user.username`.  So you maintain relational integrity and column uniqueness, but you don't maintain display uniqueness.

I don't remember what our reasoning (if any) was for having a different Mobilito name than the global name.  Perhaps it would be more useful just to use the global name and be done with it.

Reply:
Also the mobilito username is something you asked for in the original issue : https://github.com/transport-nantes/tn_web/issues/1115

I believed it was to separate mobilito from the rest ?

So what should we be doing with the username ? Only display the username in mobilito model (then we'd have no "Fait par X" string until the user set a username) ? Or we use and edit the one in User ?